src/components/DisciplineSection.vue

```diff
@@ -1,18 +1,24 @@
-<!--
- Displays entire section of disciplines
- receives events from child component and check if change is possible in resources
- if changes are possible, emit event to top component to make changes, otherwise don't
- -->
-
-<script>
+<script setup>
+/**
+ *  Displays entire section of disciplines
+ * receives events from child component and check if change is possible in resources
+ * if changes are possible, emit event to top component to make changes, otherwise don't
+ */
 import { statSectionMixin } from "./mixins/statSectionMixin";
 import Stat from "./Stat.vue";
 import RestrictionState from "./RestrictionState.vue";
 
+const props = defineProps({
```

Comment:
a ked sa tam davaju uz props, tak by mi prislo logicke ze sa tam daju emits

Reply:
no, mozno to tak funguje, nezkusal som

Reply:
hmmm, defineMixins funciu IDE nepozna, narozdiel od defineProps a defineEmits